release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -426,6 +426,18 @@ You must still use the `performance-addon-operator-must-gather` image when runni
 [id="ocp-4-11-nodes"]
 === Nodes
 
+[id="ocp-4-11-nodes-self-node-remediation"]
+==== Self Node Remediation Operator replaces the Poison Pill Operator
+{product-title} 4.11 introduces the Self Node Remediation Operator that replaces the Poison Pill Operator.
+
+The Self Node Remediation Operator provides the following enhancements:
+
+* Introduces separate remediation templates based on the remediation strategy
+* Captures last error message if remediation is unsuccessful
+* Improves metrics for Operator's configuration parameters by providing minimum values for the parameters.
+
+For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc[Remediating nodes with the Self Node Remediation Operator]
```

Comment:
the xref doesn't work on my end. I'm assuming it's because of the local build. Can you double check and ensure the xref works on your end?

Reply:
Fixed it. It was missing the the anchor tag. Thank you!